data/sql/esri.sql

```diff
@@ -6752,6 +6770,37 @@ INSERT INTO alias_name VALUES('projected_crs','EPSG','9680','WGS_84_TM_90_NE','E
 INSERT INTO alias_name VALUES('projected_crs','EPSG','9697','REDGEOMIN_UTM_zone_12S','ESRI');
 INSERT INTO alias_name VALUES('projected_crs','EPSG','9698','REDGEOMIN_UTM_zone_18S','ESRI');
 INSERT INTO alias_name VALUES('projected_crs','EPSG','9699','REDGEOMIN_UTM_zone_19S','ESRI');
+INSERT INTO "projected_crs" VALUES('ESRI','9709','NAD_1983_CSRS_UTM_Zone_23N',NULL,'EPSG','4400','EPSG','4617','EPSG','16023',NULL,0);
```

Comment:
do you know why this is promoted as a plain projected_crs object, and not just an alias to EPSG:9709  ?

Reply:
Again, it's coming from the ESRI source csvs which list ESRI as the authority for this: https://raw.githubusercontent.com/Esri/projection-engine-db-doc/master/csv/pe_list_projcs.csv


9709,9709,"PE_PCS_NAD83_CSRS_UTM_23N","NAD_1983_CSRS_UTM_Zone_23N","PROJCS[""NAD_1983_CSRS_UTM_Zone_23N"",GEOGCS[""GCS_North_American_1983_CSRS"",DATUM[""D_North_American_1983_CSRS"",SPHEROID[""GRS_1980"",6378137.0,298.257222101]],PRIMEM[""Greenwich"",0.0],UNIT[""Degree"",0.0174532925199433]],PROJECTION[""Transverse_Mercator""],PARAMETER[""False_Easting"",500000.0],PARAMETER[""False_Northing"",0.0],PARAMETER[""Central_Meridian"",-45.0],PARAMETER[""Scale_Factor"",0.9996],PARAMETER[""Latitude_Of_Origin"",0.0],UNIT[""Meter"",1.0]]","PROJCRS[""NAD_1983_CSRS_UTM_Zone_23N"",BASEGEOGCRS[""GCS_North_American_1983_CSRS"",DATUM[""D_North_American_1983_CSRS"",ELLIPSOID[""GRS_1980"",6378137.0,298.257222101,LENGTHUNIT[""Meter"",1.0]]],PRIMEM[""Greenwich"",0.0,ANGLEUNIT[""Degree"",0.0174532925199433]],CS[ellipsoidal,2],AXIS[""Latitude (lat)"",north,ORDER[1]],AXIS[""Longitude (lon)"",east,ORDER[2]],ANGLEUNIT[""Degree"",0.0174532925199433]],CONVERSION[""Transverse_Mercator"",METHOD[""Transverse_Mercator""],PARAMETER[""False_Easting"",500000.0,LENGTHUNIT[""Meter"",1.0]],PARAMETER[""False_Northing"",0.0,LENGTHUNIT[""Meter"",1.0]],PARAMETER[""Central_Meridian"",-45.0,ANGLEUNIT[""Degree"",0.0174532925199433]],PARAMETER[""Scale_Factor"",0.9996,SCALEUNIT[""Unity"",1.0]],PARAMETER[""Latitude_Of_Origin"",0.0,ANGLEUNIT[""Degree"",0.0174532925199433]]],CS[Cartesian,2],AXIS[""Easting (X)"",east,ORDER[1]],AXIS[""Northing (Y)"",north,ORDER[2]],LENGTHUNIT[""Meter"",1.0]]","NAD 1983 CSRS UTM Zone 23N","**Esri**","12.9.0","no","Canada - 48~W to 42~W",39.85,54.47,-48.0,-42.0


Want me to add logic to override this too?

Reply:
> Want me to add logic to override this too?

if that's not too complicated.

(by the way, I also noted a whole set of new projected CRSs, like ESRI:20904, being added under the ESRI authority where they were EPSG equivalent ones, with the difference that the EPSG ones had northing, easting axis order, whereas the ESRI WKT2 had an explicit easting, northing order. So having separated objects made sense. From what I see, they are new additions in the pe_list_projcs.csv, so that makes sense. Looks like Esri doesn't want to play with the axis order fun :-) Can't blame them...  )